src/app.css

```
@@ -182,19 +180,29 @@
 
 @layer utilities {
   html {
-    @apply scroll-smooth overflow-y-scroll;
+    @apply scroll-smooth;
+    scrollbar-width: thin;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)